src/nunit-gui/Views/SettingsPage.cs

```diff
@@ -56,16 +51,15 @@ public SettingsPage(string key, SettingsModel settings)
 
         #region Properties
 
+        private bool settingsLoaded;
+
         protected SettingsModel Settings { get; private set; }
 
         protected IMessageDisplay MessageDisplay { get; private set; }
 
         public string Key { get; private set; }
 
-        public bool SettingsLoaded
-        {
-            get { return true; }// _settings != null; }
-        }
+        public bool SettingsLoaded => settingsLoaded;
```

Comment:
From the comment on the old code, it looks as if the property formerly checked that _settings is not null. I can't remember why we stopped doing that. Usually, when something is commented out like that, it means I'm in the middle of working on it and made a temporary change. The change should not be in master, but it may have been committed by mistake.